helix-term/src/commands.rs

```diff
@@ -506,12 +515,16 @@ impl PartialEq for MappableCommand {
         match (self, other) {
             (
                 MappableCommand::Typable {
-                    name: first_name, ..
+                    name: first_name,
+                    args: first_args,
+                    ..
                 },
                 MappableCommand::Typable {
-                    name: second_name, ..
+                    name: second_name,
+                    args: second_args,
+                    ..
                 },
-            ) => first_name == second_name,
+            ) => first_name == second_name && first_args == second_args,
```

Comment:
This was necessary to make the test check whether the argument was parsed correctly. I did not consider whether this would change any other behaviour

Reply:
I don't think this should have any runtime impact. AFAIK PartialEq is only used on this enum for test cases and for the keymap test cases